Scan/STLcd/DOS_8x8.h

```diff
@@ -0,0 +1,149 @@
+/* "DOS 8x8" font by Neil Roy
```

Comment:
This commit is awesome, and a good start for getting LCD font rendering.

I'm trying to decide how best to store fonts/symbols on the github side of things. Nothing wrong with how you did it, as it's similar (if not cleaner), than what I have in the capabilities.kll file for images.

In the future I'm planning on support different displays, perhaps even doing font rendering across the backlit switch LEDs. While I'm still finalizing how this abstraction would will look (that's part of what KLL 0.5 is for), I think it should be possible to store fonts (at least on github) in a more universal way.

So, my idea:
- Store the original bitmap font on github
- During the compilation (perhaps a sub-module of kll), convert the font to the desired format (i.e. what you have below).
- Since different displays have different ways of storing data, this compilation step code optimize for that particular display
- Should also be easier to manipulate fonts with standard font tools, rather than editing the Hex directly
- With a bit more work, could be extended to non-bitmap fonts

Thoughts?
(Also, I'm in no way I'm asking you to work on this; though if you want to... :P )